src/pnpq/devices/utils.py

```diff
@@ -0,0 +1,28 @@
+from serial.tools.list_ports import comports as list_comports
+import logging
+
+logger = logging.getLogger("utils")
+
+USB_HUB_iVENDOR = "2109"
+USB_HUB_iPROFUCT = ["0817", "2817"]
```

Comment:
the number of available usb hubs will grow. so, we should check by the tuple of vid and pid.
then the available usb hubs are defined like this
```py
AVAILABLE_USB_HUBS: list[tuple[str, str]] = [
  ("2109", "0817") # some usb hub name
  ("2109", "2817") # another usb hub name
]
```